test-charts/headless-service/advanced-scaling/values.yaml

```diff
@@ -24,3 +24,8 @@ bandstand-web-service:
         desiredReplicas: 1
         start: 0 7 * * MON-FRI
         end: 0 19 * * MON-FRI
+      - type: kafka
+        topic: event.msk.reference.dummy
+        consumerGroup: keda-consumer
+        bootstrapServersEnvVar: MSK_BOOTSTRAP_SERVERS # optional - defaults to KAFKA_BROKERS
+        tls: enabled
```

Comment:
Whilst it doesn't really matter for the purpose of the test would be better if these examples used valid values 'enable' or 'disable'